app/views/users/new.html.erb

```diff
@@ -1,52 +1,113 @@
+        <%= f.email_field :email, :help => t(".email_help_html",
+                                             :privacy_policy_link => link_to(t(".privacy_policy"),
+                                                                             t(".privacy_policy_url"),
+                                                                             :title => t(".privacy_policy_title"),
+                                                                             :target => :new)),
+                                  :tabindex => 1 %>
+      <% else %>
+        <%= f.hidden_field :email %>
+      <% end %>
 
-      <%= f.text_field :display_name, :help => t(".display name description"), :tabindex => 3 %>
+      <%= f.text_field :display_name, :help => t(".display name description"), :tabindex => 2 %>
 
-      <fieldset class="mb-3" id="auth_field">
-        <label for="user_auth_provider" class="form-label"><%= t(".external auth") %></label>
+      <% if current_user.auth_uid.nil? %>
         <div class="row">
-          <%= f.select(:auth_provider, Auth.providers, :default => "", :hide_label => true, :wrapper => { :class => "col-auto mb-0" }, :tabindex => 4) %>
-          <%= f.text_field(:auth_uid, :hide_label => true, :wrapper => { :class => "col mb-0" }, :tabindex => 5) %>
+          <div class="col-sm">
+            <%= f.password_field :pass_crypt, :tabindex => 3 %>
+          </div>
+          <div class="col-sm">
+            <%= f.password_field :pass_crypt_confirmation, :tabindex => 4 %>
+          </div>
         </div>
-        <small class="form-text text-muted"><%= t ".auth no password" %></small>
-      </fieldset>
+      <% end %>
 
-      <%= f.password_field :pass_crypt, :tabindex => 6 %>
-      <%= f.password_field :pass_crypt_confirmation, :tabindex => 7 %>
+      <p class="mb-3 text-muted fs-6"><%= t(".by_signing_up_html",
+                                            :tou_link => link_to(t(".tou"),
+                                                                 "https://wiki.osmfoundation.org/wiki/Terms_of_Use",
+                                                                 :target => :new),
+                                            :privacy_policy_link => link_to(t(".privacy_policy"),
+                                                                            t(".privacy_policy_url"),
+                                                                            :title => t(".privacy_policy_title"),
+                                                                            :target => :new),
+                                            :contributor_terms_link => link_to(t(".contributor_terms"),
+                                                                               t(".contributor_terms_url"),
+                                                                               :target => :new)) %></p>
+      <%= f.form_group do %>
+        <%= f.check_box :consider_pd,
```

Comment:
I suspect that when this checkbox is placed here, people will tick it without reading what is says thinking that it's necessary to go forward.

Reply:
> You might be referring to TOU and privacy policy

That's what users will think this checkbox is about, but it's not.